src/confd/scripts/setup.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Add a note about the origin of this script (netopeer?) so that it's easier for the next person to find.